src/crc32_fast.rs

```diff
@@ -0,0 +1,226 @@
+use super::{Algorithm, Crc, Digest};
+use crate::table::crc32_table_slice_16;
+
+/// This implements a faster version of Crc<u32> with a 16kB lookup table
+pub struct FastU32;
```

Comment:
Maybe make this generic over width (e.g. `Fast<u32>`)? The `impl` doesn't have to be generic, but it leaves open that possibility in case const fns in traits are stable.

Reply:
Done